### PR TITLE
Build raxml for osx-arm64

### PR DESCRIPTION
**Pull request by corneliusroemer**

### CODE REVIEW

**Comment by corneliusroemer** [comment]
@martin-g I noticed you upstreamed some changes to raxml to add aarch64 support. Maybe you could do likewise here? I'm using a patch for now as I don't understand the details of the necessary changes, but would be cool if you could have a look at whether we might build with pthreads etc for arm64 as well.

**Comment by corneliusroemer** [comment]
@BiocondaBot please add label

**Comment by corneliusroemer** [comment]
@BiocondaBot please fetch artifacts

File: recipes/raxml/build.sh [hunk 1]
```diff
@@ -24,8 +33,8 @@ for PTHREADS in "" .PTHREADS; do
     else
       MAKEFILE=${MAKEFILE}.gcc
     fi
-    make -f ${MAKEFILE} CC=$CC
```
Comment (martin-g) [inline recipes/raxml/build.sh:27]: Why did you remove `CC=$CC` ?
I think this is the reason why now you need the `patch` (to remove `CC = gcc`.

**Comment by corneliusroemer** [comment]
Ah this is a great point! I don't quite remember, maybe I was trying to
debug locally and didn't have the CC env variable set. I don't know very
much about make yet.

Thanks for your eyes!

File: recipes/raxml/build.sh (hunk 1 above)
Comment (corneliusroemer) [reply recipes/raxml/build.sh:27]: 🤦 stupid me, thanks for noticing!

**Review by martin-g — APPROVED** [review]
